opencensus/proto/export/BUILD.bazel

```diff
@@ -0,0 +1,34 @@
+# Copyright 2018, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+package(default_visibility = ["//visibility:public"])
+
+proto_library(
+    name = "export_proto",
+    srcs = ["export.proto"],
+    deps = [
+        "@com_google_protobuf//:duration_proto",
```

Comment:
You don't need this correct? same for timestamp.

You should add dependency on the metric_proto and span_proto.

Reply:
Done.